### DISCUSSION TITLE
Tracking: Alternate Icon Ideas

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5539] [Ideas]
This is a tracking issue for the second push on alternate icons. The first set of two icons is going out in v2.5. 

## Iconographical Ideas
- [x] Windmill
- [ ] MGM Lion (@hawkrives)
- [ ] Squirrel (@drewvolz)
- [ ] Pizza (@drewvolz)
- [ ] Water tower (lion silhouette on tower top) (@hawkrives)
- [ ] Circuit lion (@drewvolz)
- [ ] Boe Chapel structure (@hawkrives)
- [ ] "O" for Olaf (@drewvolz)
- [ ] UmYahYah square from new branding guidelines (@hawkrives)
- [ ] Something calligraphical/fancy? (@rye)

## Other Ideas
- Offer each as a full/multi-color basic icon
- Also offer six-ish other single-color-tint variants, for people who like to colorize their homescreens 
    - blue
    - red
    - green
    - gold
    - purple
    - pink

**Comment by drewvolz** [comment]
<details>
<summary>Circuit lion</summary>
<br>
<img src="https://avatars3.githubusercontent.com/u/29408373?s=200&v=4" />
</details>

**Comment by drewvolz** [comment]
I really enjoy [The Noun Project](https://thenounproject.com/)'s set of icons. We can even use them freely if we credit.

- [Water tower](https://thenounproject.com/search/?q=water%20tower)
- [Chapel](https://thenounproject.com/search/?q=chapel)
- [Pizza](https://thenounproject.com/search/?q=pizza)
- [Squirrel](https://thenounproject.com/search/?q=squirrel)

**Comment by hawkrives** [comment]
https://bl.ocks.org/hawkrives/raw/52076be27ca1bde0c0b83524ed8cd7bf/

I've got part of a skeleton for generating colorized versions of our icons; this will be even nicer when we have them as vectors, but this POC shows that we needen't always impose that restriction. :)

<details><summary>For longevity:</summary>

![result-oldmain](https://user-images.githubusercontent.com/464441/36069493-87e13338-0eb0-11e8-8d97-0bac5b959ad5.png)
![result-windmill](https://user-images.githubusercontent.com/464441/36069494-87f9440a-0eb0-11e8-96a3-b915db02d019.png)

</details>

<br/>

References for `feColorMatrix`: 

- https://stackoverflow.com/questions/14663252/matching-a-color-in-svg-with-fecolormatrix#14663343
- https://css-tricks.com/color-filters-can-turn-your-gray-skies-blue/
- http://paletton.com/#uid=13B0u0kwi++bu++hX++++rd++kX (for colors)
- https://stackoverflow.com/questions/21977929/match-colors-in-fecolormatrix-filter#21979654
- http://alistapart.com/article/finessing-fecolormatrix – I may need to re-read this one to get any value out of it

**Comment by hawkrives** [comment]
I found the "midjourney" ml bot (by way of Steve T-S). Here are some examples:

• | • | • | • 
---|---|---|---
![image](https://user-images.githubusercontent.com/464441/180426776-2943c443-579e-4f7c-8d2b-3d4e443a0d7b.jpeg) | ![image](https://user-images.githubusercontent.com/464441/180426808-7bd533e4-b34f-4e8a-9717-45ba3046dc5a.png) | ![image](https://user-images.githubusercontent.com/464441/180427146-fe22837f-254e-441a-9c4a-30a58fa0d7cd.jpeg) | ![image](https://user-images.githubusercontent.com/464441/180427164-6798b900-4347-419c-8a55-03fbba8ac792.jpeg)
![image](https://user-images.githubusercontent.com/464441/180427200-ceecb011-f877-4bc8-b25e-a66649e09842.jpeg) | ![IMG_0985](https://user-images.githubusercontent.com/464441/180427439-b8565252-2185-4a35-8607-871af4f7ac8c.png) | ![IMG_0993](https://user-images.githubusercontent.com/464441/180427458-c9e382e5-2ed0-4d2d-8ab1-7460cd922212.png) | ![IMG_0994](https://user-images.githubusercontent.com/464441/180427463-2830b971-3c1c-454e-b530-f7606a006d05.png)
![IMG_0995](https://user-images.githubusercontent.com/464441/180427464-2e565e51-7779-43c0-8d48-7359875cd240.png) | ![IMG_0997](https://user-images.githubusercontent.com/464441/180427469-a0aac7df-c29d-4c35-aae0-a4e36f387cd0.png) | ![IMG_0998](https://user-images.githubusercontent.com/464441/180427471-42557fea-6839-4658-99c1-da8f75b5a238.png) | ![IMG_0999](https://user-images.githubusercontent.com/464441/180427472-ba5c8aed-3cf9-4e71-a632-8fc1e0f27955.png)
![hawkrives_realistic_iOS_app_icon_depicting_a_Norse_lion_c000df1f-7ec5-45f9-a654-cde1e72c79d6](https://user-images.githubusercontent.com/464441/180427474-efd07252-5efb-41dc-9a8c-7e6db4a177b8.png) | ![hawkrives_realistic_iOS_app_icon_depicting_a_Norse_lion_028107e5-04a5-4c7b-ab5e-7d6975fc4ca4](https://user-images.githubusercontent.com/464441/180427476-d496cd43-3d78-449f-8fc4-6594d7bbad7f.png) | ![hawkrives_realistic_iOS_app_icon_depicting_a_Norse_lion_55d93fc5-733c-4c7d-a234-acf68a140299](https://user-images.githubusercontent.com/464441/180427477-7a7867c8-bb22-48b5-afc1-c5684b300512.png) | ![hawkrives_a_realistic_iOS_icon_with_a_lion_Art_Deco_afd7b4c8-cefe-4459-ad11-eb01abe61db1](https://user-images.githubusercontent.com/464441/180427479-ae49b213-513a-42b4-a1b6-af22129188e4.png)
![IMG_1018](https://user-images.githubusercontent.com/464441/180427480-4a3d03da-5e8a-4ca3-980c-846938d0e804.png) | ![IMG_1019](https://user-images.githubusercontent.com/464441/180427483-b0ce213f-c36a-4fb6-91aa-bee46815c9cb.png) | ![IMG_1020](https://user-images.githubusercontent.com/464441/180427484-b3070aec-3d86-4113-ac6d-1dc8cc794093.png) | ![IMG_1021](https://user-images.githubusercontent.com/464441/180427485-fb416083-a415-4031-8ab3-ab1df2cc1bc9.png)
![IMG_1022](https://user-images.githubusercontent.com/464441/180427486-07a03a45-e7df-4bbf-8b73-795c0bd8ee3b.png) | ![IMG_1023](https://user-images.githubusercontent.com/464441/180427491-ca19757d-09ae-4feb-8774-7abc009e487d.png) | ![IMG_1024](https://user-images.githubusercontent.com/464441/180427494-a743e15d-1c2a-4375-a4d8-586ff993cf42.png) | ![hawkrives_an_iOS_icon_with_an_Art_Deco_lion_b01057f2-2a62-4c1c-b16d-2d2b1dad43d0](https://user-images.githubusercontent.com/464441/180427496-ebd9ec18-b85b-4125-abef-f177dae304d3.png)
![Jowvi_a_realistic_iOS_icon_with_a_lion_Art_Deco_6eed4481-17c8-4a28-9134-169e3dacad18](https://user-images.githubusercontent.com/464441/180427497-0786c939-6870-430b-9aa3-247af7a02606.png) | ![IMG_1028](https://user-images.githubusercontent.com/464441/180427498-62509964-b5eb-4529-863e-197d63d1a5a1.png) | ![hawkrives_a_realistic_iOS_app_icon_with_a_lion_geometric_a1e1cab0-0817-431f-b76f-9cde29483fc7](https://user-images.githubusercontent.com/464441/180427499-3580d74b-3187-46fa-9d0c-6e751038ac1d.png) | ![hawkrives_an_iOS_app_icon_with_a_photo-realistic_lion_6802dea9-3aa1-432c-84e1-317ecf5a69cf](https://user-images.githubusercontent.com/464441/180427501-261cb684-1971-4b7b-b9c5-374a5658c420.png)
![hawkrives_an_iOS_app_icon_with_a_skeuomorphic_lion_2c08e4f4-8ee3-4398-ac86-a60485430943](https://user-images.githubusercontent.com/464441/180427502-94db9dd9-0b24-4b29-9104-5af64a690d55.png) | ![hawkrives_a_realistic_iOS_app_icon_with_a_windmill_in_front_of__d3f04378-2290-498f-9f81-b302ffe20734](https://user-images.githubusercontent.com/464441/180427505-9f2982da-4491-4fbd-8e5d-e0b5bb16804f.png) | ![hawkrives_an_iOS_app_icon_with_a_windmill_set_against_an_evenin_dc8269b8-f799-4bfb-94c4-fce58fa3114a](https://user-images.githubusercontent.com/464441/180427507-3b3a2f1e-5b97-4831-9464-07c4f04e4ae6.png) | ![IMG_0986](https://user-images.githubusercontent.com/464441/180427447-361c31be-a385-401b-be03-54afaa3c660b.png)
![IMG_0987](https://user-images.githubusercontent.com/464441/180427448-0c41a302-f9d2-4efd-ab74-46704ed7d991.png) | ![hawkrives_realistic_detailed_iOS_app_icon_depicting_a_lion_a71cd54f-f789-4ca7-b3ad-6d58e80ee4d0](https://user-images.githubusercontent.com/464441/180427451-d09e8cca-59b0-4503-9340-1b28e3a32cae.png) | ![hawkrives_realistic_detailed_iOS_app_icon_depicting_a_lion_ea486df9-1762-44dd-9013-486c77fc0f60](https://user-images.githubusercontent.com/464441/180427453-72245f00-f77f-4c75-ba62-4e8524c32b8c.png) | ![hawkrives_iOS_app_icon_depicting_a_lion_ukio-e_happy_6610a3b6-e028-4ccf-8616-18e9506025ca](https://user-images.githubusercontent.com/464441/180427454-fa107986-822f-4cf9-a7cb-08c9aa74c76c.png)
![IMG_0992](https://user-images.githubusercontent.com/464441/180427456-4ecc5817-32a5-45c1-9c58-5d80d07e5bc8.png)

**Reply by hawkrives** [reply]
I'll come through and cleaning up this post into a thread, for easier voting and display, and replacing the screenshots with better quality PNGs later today.
